README.md

```diff
@@ -152,6 +152,33 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 }
 ```
 
+### RationaleCallbacks
```

Comment:
Nit: new line after. Also, how about calling it something like "Interacting with the rationale dialog"?

Reply:
👍